test/unit/batching/vendor-lines/get-vendor-line.test.js

```diff
@@ -1,7 +1,7 @@
-const { getVendorLineAP, getVendorLineAR } = require('../../../app/batching/get-vendor-line')
-const { EUR } = require('../../../app/constants/currency')
-const { AR } = require('../../../app/constants/ledgers')
-const { NOT_APPLICABLE } = require('../../../app/constants/not-applicable')
+const { getVendorLineAP, getVendorLineAR } = require('../../../../app/batching/vendor-lines/get-vendor-line')
+const { EUR } = require('../../../../app/constants/currency')
+const { AR } = require('../../../../app/constants/ledgers')
+const { NOT_APPLICABLE } = require('../../../../app/constants/not-applicable')
```

Comment:
I think the constants should be above the definitions for the vendor lines

Reply:
Agree, I haven't changed any of these tests, just moved the folder they were in.

Updated this one as it's a quick win